src/cmd/linuxkit/push_gcp.go

```diff
@@ -5,39 +5,38 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
+	"strings"
 
 	log "github.com/Sirupsen/logrus"
 )
 
-// Process the run arguments and execute run
 func pushGcp(args []string) {
-	gcpCmd := flag.NewFlagSet("gcp", flag.ExitOnError)
+	flags := flag.NewFlagSet("gcp", flag.ExitOnError)
 	invoked := filepath.Base(os.Args[0])
-	gcpCmd.Usage = func() {
-		fmt.Printf("USAGE: %s push gcp [options] [name]\n\n", invoked)
-		fmt.Printf("'name' specifies the full path of an image file which will be uploaded\n")
+	flags.Usage = func() {
+		fmt.Printf("USAGE: %s push gcp [options] path\n\n", invoked)
```

Comment:
should this be `GCS` instead of `GS`?  I'm not too familiar with their terminology